bootstrap/bootstrapper.go

```diff
@@ -3,14 +3,8 @@ package bootstrap
 // DO NOT PUT any app logic in this package to avoid any dependency cycles
 
 const (
-	BootstrappedConfigFile       string = "BootstrappedConfigFile"
-	BootstrappedConfig           string = "BootstrappedConfig"
-	BootstrappedLevelDb          string = "BootstrappedLevelDb"
-	BootstrappedEthereumClient   string = "BootstrappedEthereumClient"
-	BootstrappedAnchorRepository string = "BootstrappedAnchorRepository"
-	BootstrappedP2PClient        string = "BootstrappedP2PClient"
-	BootstrappedP2PServer        string = "BootstrappedP2PServer"
-	BootstrappedAPIServer        string = "BootstrappedAPIServer"
+	BootstrappedP2PServer string = "BootstrappedP2PServer"
```

Comment:
Moved most of the bootstrapped objects to its ow packages, except those that are used commonly.